compiler/aarch64/codegen/OMRCodeGenerator.cpp

```diff
@@ -625,3 +625,9 @@ OMR::ARM64::CodeGenerator::generateScratchRegisterManager(int32_t capacity)
    {
    return new (self()->trHeapMemory()) TR_ARM64ScratchRegisterManager(capacity, self());
    }
+
+bool
+OMR::ARM64::CodeGenerator::enableRegisterAssociations()
+   {
+   return true;
```

Comment:
Can you move this function body to OMRCodeGenerator.hpp if it only returns `true`.

Reply:
Moved it to OMRCodeGenerator.hpp.